src/main/java/io/github/thebusybiscuit/sensibletoolbox/items/PaintBrush.java

```diff
@@ -41,6 +40,7 @@
 import io.github.thebusybiscuit.sensibletoolbox.utils.STBUtil;
 import io.github.thebusybiscuit.sensibletoolbox.utils.UnicodeSymbol;
 import me.desht.dhutils.Debugger;
+import me.mrCookieSlime.CSCoreLibPlugin.general.Inventory.ChestMenu;
```

Comment:
CS-CoreLib is no longer supported

Reply:
Ye, I tried to implement GUIUtil/InventoryGUI but InventoryCloseEvent is getting missfired when executing "gui.show(player)" and STB is losing track of opened inventory. And because of this anomaly GUIUtil's custom inventory is not working and player is able to remove items from it.
I tried to dump event stacktrace and it looks that event is executed because of received packet but player have just opened the inventory. But everything is working fine for other ui's.
Idk why clicking image with a shovel is causing client to send inventoryclosed packet...

![obraz](https://user-images.githubusercontent.com/25874220/142753920-f130fa6a-2190-41a1-9e98-729d8bbedf25.png)
Result: Player is able to remove items from inventory, gadgets're not working

Reply:
Done. Also fixed block painting.